types/lodash/index.d.ts

```diff
@@ -7221,296 +7257,879 @@ declare namespace _ {
         ): T|undefined;
     }
 
-    interface LoDashExplicitWrapperBase<T, TWrapper> {
+    interface LoDashExplicitWrapperBase<T, TWrapper> {
+        /**
+         * @see _.findLast
+         */
+        findLast(
```

Comment:
@andy-ms because it's an explicit wrapper - it doesn't return `T` (nor `undefined`) - it returns another explicit wrapper. The problem is that we don't know what type of explicit wrapper to return (explicit array wrapper? explicit dictionary wrapper? explicit string wrapper?). See https://lodash.com/docs/4.17.4#chain.

For example:

```
_.chain([[1, 2, 3], [4, 5, 6], [7, 8]])
    .findLast((a) => a.length === 3) // should return LodashExplicitArrayWrapper<number>, since T is number[]
    .value(); // now this returns number[]
```

The problem is we don't know if `T` is an array, dictionary, or something else, so we don't know which type of wrapper to return. So we just return `any`, and let the caller cast it to the correct type.